exchange/adapter_util.go

```diff
@@ -43,6 +43,11 @@ func buildBidders(infos config.BidderInfos, builders map[openrtb_ext.BidderName]
 			continue
 		}
 
+		if err := SetAliasBuilder(info, builders, bidderName); err != nil {
+			errs = append(errs, fmt.Errorf("%v: failed to set alias builder: %v", bidder, err))
+			continue
+		}
+
 		builder, builderFound := builders[bidderName]
 		if !builderFound {
 			errs = append(errs, fmt.Errorf("%v: builder not registered", bidder))
```

Comment:
```
func SetAliasBuilder(info config.BidderInfo, builders map[openrtb_ext.BidderName]adapters.Builder, bidderName openrtb_ext.BidderName) error {

        ....
	L81: builders[bidderName] = builder <------------------ builder assigned for alias adapter
	L82: return nil
}

```

`SetAliasBuilder()` adds builder to `builders` map for an alias (line 81). so check from line 51-53 can be skipped in case of alias adapter.

Reply:
Or protect the call to setAliasBuilder with `if` clause like - 

if len(info.AliasOf) > 0 {
  //setAliasBuilder
} else {
 //set the concrete builder.
}

Reply:
added check in [9dc7c5b](https://github.com/prebid/prebid-server/pull/3016/commits/9dc7c5b8aeb3a23fa9972777820738a08ec1f4f4)